synapseclient/core/upload/upload_functions.py

```diff
@@ -195,17 +195,18 @@ def create_external_file_handle(syn, path, mimetype=None, md5=None, file_size=No
     url = as_url(os.path.expandvars(os.path.expanduser(path)))
     if is_url(url):
         parsed_url = urllib_parse.urlparse(url)
-        if parsed_url.scheme == "file" and os.path.isfile(parsed_url.path):
-            actual_md5 = md5_for_file(parsed_url.path).hexdigest()
+        parsed_path = file_url_to_path(url)
```

Comment:
This is a Windows specific bug that I found with the integration tests I wrote for my new logic. The `parsed_url.path` was showing up as `/c:/WINDOWS/asdf.txt` which was preventing `isfile` and all subsequent code from getting the correct data.

No errors were showing up - but that means we haven't been storing the MD5 checksum or file size into Synapse on windows for any local files.